release-notes/swan-lake-beta1-release-note.md

```diff
@@ -55,6 +78,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github
 
 #### Improvements
 
+#### Breaking Changes
+
+- `io.ballerina.runtime.api.types.Type#getName` and `io.ballerina.runtime.api.types.Type#getQualifiedName` now returns an empty string if no name was associated with the type. The `io.ballerina.runtime.api.types.Type#toString` method can be used to get the string representation of a type if required.
+- Wait actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error`, `T` being the type of the original return value.
```

Comment:
```suggestion
- The` wait` actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error`in which `T` being the type of the original return value.
```

Reply:
Any particular reason to change from "Wait actions"? Do we really need the "the"?

Also isn't "where" more suitable that "in which". The spec also uses where.

```suggestion
- Wait actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error`, where `T` is the type of the original return value.
```

Reply:
+1